it/http/src/test/resources/com/adobe/cq/commerce/it/http/magento-graphql-category-tree-2.3.0.json

```diff
@@ -0,0 +1,182 @@
+{
```

Comment:
We should use the new `2.3.1` version of the JSON response, I already changed that also in the unit tests.

Reply:
I had a 2nd thought about that, and the fact that the ITs pass with this old `2.3.0` category response and because I removed the code that fixed this temporarily, I think that the ITs are not testing that the categories are properly displayed in the console. We should keep this in mind and add some better tests at some point.